### PR TITLE
Bootstrap Basic UI using React

### LINKED ISSUE #17: Bootstrap Basic UI using React

### PR DESCRIPTION
**Pull request by rasmitha-koduru**
relates to #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +0,0 @@
-## Team CAPtivate
```
Comment (adithyaselv) [inline README.md]: please revert this change

File: docs/contributing.md [hunk 2]
```diff
@@ -1,54 +0,0 @@
-### Contributing Guideline
```
Comment (adithyaselv) [inline docs/contributing.md]: please revert this change

File: src/setupTests.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
```
Comment (adithyaselv) [inline src/setupTests.js]: why do we need this ?

**Review by adithyaselv — CHANGES_REQUESTED** [review]
Please make requested changes and add `package-lock` file to gitignore

File: src/setupTests.js (hunk 3 above)
Comment (rasmitha-koduru) [reply src/setupTests.js]: This is under the folder structure of create-react-app package. We can delete or rename these  files. I'm planning to remove and test locally and update it for the next version.

File: README.md (hunk 1 above)
Comment (rasmitha-koduru) [reply README.md]: Reverted, thanks!

File: docs/contributing.md (hunk 2 above)
Comment (rasmitha-koduru) [reply docs/contributing.md]: Added file

File: src/setupTests.js (hunk 3 above)
Comment (adithyaselv) [reply src/setupTests.js]: thanks for clarifying

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+package-lock.json
+
+npm-debug.log*
+yarn-debug.log*
```
Comment (adithyaselv) [inline .gitignore]: are we using npm or yarn ?

Comment (rasmitha-koduru) [reply .gitignore]: npm yes, yarn i dont think so.

Comment (adithyaselv) [reply .gitignore]: okay please remove this and also resolve the merge conflict

**Review by adithyaselv — CHANGES_REQUESTED** [review]
Hey ! just noticed! as we discussed earlier, could move all the changes related to UI in the `UI` or `WeatherApp` folder ?

**Review by adithyaselv — APPROVED** [review]
Good first PR
